lib/ecto_adapters_dynamodb.ex

```diff
@@ -48,38 +37,34 @@ defmodule Ecto.Adapters.DynamoDB do
     }
 
     # Pass some config values through to ex_aws.
```

Comment:
If we no longer pass the config through to ex_aws here, can we remove this comment?

Reply:
Indeed we can.